### ISSUE TITLE
Non-grouped panels show up as grouped when nested inside a grouped panel

### ISSUE THREAD

**Issue by timscaffidi** [twbs/bootstrap#17636]
non-grouped panels that are inside of a panel that is itself part of a panel group are getting the margin-bottom:0px style applied. This makes them appear as though they are grouped even when they are not, and messes up spacing with anything that comes below them.

See this plunker for an example with a work-around: http://plnkr.co/edit/9i94bBFeWCIusPiuVhf6?p=preview

**Comment by twbs-lmvtfy** [comment]
Hi @timscaffidi!

You appear to have posted a live example (http://run.plnkr.co/plunks/9i94bBFeWCIusPiuVhf6/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W001](https://github.com/twbs/bootlint/wiki/W001): `<head>` is missing UTF-8 charset `<meta>` tag
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness
* line 5, column 5: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.5 ; saw what appears to be usage of Bootstrap 3.3.1

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by timscaffidi** [comment]
fixed bootlint errors. Updated plunker: http://plnkr.co/edit/9i94bBFeWCIusPiuVhf6?p=preview

**Comment by twbs-lmvtfy** [comment]
Hi @timscaffidi!

You appear to have posted a live example (http://run.plnkr.co/plunks/9i94bBFeWCIusPiuVhf6/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W001](https://github.com/twbs/bootlint/wiki/W001): `<head>` is missing UTF-8 charset `<meta>` tag

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by timscaffidi** [comment]
fixed bootlint errors. Updated plunker: http://plnkr.co/edit/9i94bBFeWCIusPiuVhf6?p=preview

**Comment by 807811b0-1373-11e5-b60b-1697f925ec7b** [comment]
I noticed this issue when using UI Bootstrap accordion component with panels nested inside the accordion panes.

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
